doc/sphinx/conf.py

```diff
@@ -0,0 +1,91 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+from pathlib import Path
+from typing import Sequence
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = "GPflow"
+copyright = "2022, James Hensman and Alexander G. de G. Matthews and many others"
+author = "James Hensman and Alexander G. de G. Matthews and many others"
+
+# The full version, including alpha/beta/rc tags
+release = "2.4.0"
```

Comment:
Can we also update the [release readme](https://github.com/GPflow/GPflow/blob/develop/RELEASE.md) with which files we need to update when making a new release?